src/app.js

```diff
@@ -51,14 +51,43 @@ export class App {
     return true;
   }
 
-  sendUserCreated(email, username, validationPolicy) {
-    if (this.emailService) {
-      // TODO
+  async sendUserCreated(email, username, validationPolicy, validationParams) {
```

Comment:
Missing real messages.

We shouldn't have  hard coded text string here for mail - Maybe we should use templates ?